src/components/shared/ObjectInspector.js

```diff
@@ -139,43 +138,46 @@ class ObjectInspector extends Component {
       objectValue = Rep({ object, mode: MODE.TINY });
     }
 
-    return dom.div(
-      {
-        className: classnames("node object-node", {
+    const onClick = e => {
+      e.stopPropagation();
+      setExpanded(item, !expanded);
+    };
+
+    const onDoubleClick = e => {
+      e.stopPropagation();
+      this.props.onDoubleClick(item, {
+        depth,
+        focused,
+        expanded
+      });
+    };
+
+    return (
+      <div
+        className={classnames("node object-node", {
```

Comment:
I could definitely have pulled this inline className (and Svg below) to a variable, but I wasn't too sure on proper naming. If you have any suggestions and want to avoid inlining, just let me know and I'll update this PR.